backend/insert_db_data.py

```diff
@@ -53,17 +53,17 @@
 
 
 def db_insert_data():
-    db.session.add_all([role, role1, role2,
-                        category, category1, category2,
-                        status1, status2, status3,
-                        user1, user2, user3,
-                        issue1, issue2, issue3,
-                        issueHistory1, issueHistory2, issueHistory3,
-                        issueHistory4, issueHistory5
+    db.session.add_all([#role, role1, role2,
+                        #category, category1, category2,
+                        #status1, status2, status3,
+                        user1, user2, user3
+                        #issue1, issue2, issue3,
+                        #issueHistory1, issueHistory2, issueHistory3,
+                        #issueHistory4, issueHistory5
```

Comment:
What is the reasons of those changes?

Reply:
fixed, just have been focused on users, sorry